---
layout: page
title: Research
---

Research in the Grant lab focuses on the development and application of state-of-the-art computational and theoretical techniques to investigate the structure, dynamics and interactions of biomedically important proteins.

At the interface of biochemistry, biophysics and bioinformatics the unique insight gained from our research is tightly coupled to a wide range of biochemical and biophysical experiments. 

<img src="{{ "/img/Fig2_venn_whiteBCK.png" | prepend: site.baseurl }}" alt="Research Intersection" style="float:left;margin-right: 15px;margin-top: 10px;heigth:245px;width:245px" >

Using this approach we have recently developed a new class of allosteric Ras & Rho molecular switch inhibitors, reported the first rationally designed faster velocity kinesin motors, demonstrated for the first time that kinesin processivity can be modulated by rational mutation of the motor domain, discovered key atomistic detriments of allosteric activation in G proteins, and developed the Bio3D software package used by thousands of researchers around the world.  

Our research goals are to advance these efforts to further facilitate the fields of protein design and drug discovery.

<br/>
<a name="motors"></a>

### [Molecular Motors & Molecular Switches](#motors)

<h5>Deciphering the physical mechanisms underlying molecular motor and switch function and how their dysfunction is related to disease.</h5>

A major portion of our work asks about the common and divergent molecular mechanisms of kinesin motors and homologous G proteins switches.  These mechanisms drive the internal coordination, communication and self-organization of eukaryotic cells; and their aberrant activities underlie a wide range of human cancers and neurodegenerative disorders.  


<img src="{{ "/img/cell.jpg" | prepend: site.baseurl }}" alt="Example switches and motors" style="float:left;margin-right:20px;margin-top:15px;height:375px" >

The molecular motors and switches that we focus on comprise a set of related nucleotide binding proteins that coordinate important cell processes including signaling, trafficking, transport, and protein synthesis. Our research aims to decipher the [structural dynamic mechanisms](http://thegrantlab.org/research/research_overview/) by which these fascinating natural nanomachines function and how their dysfunction is related to disease.  

**Our recent findings on these systems include how to rationally engineer faster velocity longer running molecular motors and how to both allosterically activate and inhibit molecular switches that are malfunctioning in more than 20% of all human cancers.**


<a href="{{ "research_overview/" | prepend: site.baseurl }}">Read more &rarr;</a>

<br/>
<a name="bioinf"></a>

### [Methods for Structural Bioinformatics & Computer Aided Drug Design](#bioinf)

<h5>Integrating protein structural dynamics, evolutionary analysis & structure based drug discovery.</h5>

Proteins are structurally flexible molecules that often depend on their coordinated dynamic fluctuations to function properly. However, most current drug development strategies take one or a few rigid protein structures and design a single molecule to block its activity (*i.e.*, catalytic pocket). We are developing a more sophisticated methodology that exploits flexible target structures, allosteric pocket identification and evolutionary analysis. 

<iframe src="https://player.vimeo.com/video/27214521?autoplay=1&loop=1&title=0&byline=0&portrait=0" width="800" height="235" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>

To this aim, our lab introduced [Bio3D](http://thegrantlab.org/bio3d/index.php), a widely used open source R package for the analysis of protein sequence-structure-dynamics relationships. In particular, Bio3D is widely used by researchers around the world for combining information from all available experimental structures and sequences with that predicted by theory and computations.  

<img src="http://thegrantlab.org/research/files/still.jpg" alt="Prediction of drugable sites on Ras" style="float:right;margin-left: 10px;margin-top: 15px" >
Motivated by the wide utility and use of Bio3D, together with our own research questions, we are currently working to extend functionality and interoperability with leading cheminformatics resources. This will enable the inclusion of protein structural flexibility and allosteric pocket detection into structure based drug design protocols. 

<h5>Merging these new methods in a single integrated environment represents an important advance that will greatly facilitate both  bioinformatics research and the design of selective inhibitors.</h5>

<a href="http://thegrantlab.org/bio3d/index.php">Read more &rarr;</a>


<br/>
<a name="pubs"></a>

### [Recent Publications](#pubs)


*   [Neck linker docking is critical for Kinesin- 1 force generation in cells but at a cost to motor speed and processivity](https://elifesciences.org/articles/44146).  
    Budaitis, **Jariwala**, Reinemann, Schimert, **Scarabelli**, **Grant**, Sept, Lang and Verhey (2019)  
    _eLife_ 8: e44146. DOI: https://doi.org/10.7554/eLife.44146
    _PLoS Comput Biol._ 14(11): e1006364.         
    ( [In Press](https://elifesciences.org/articles/44146) )  

<br/>

*   [Comparative protein structure analysis with Bio3D-web]().  
    **Grant**, Skjærven and Yao (2019)  
    _Methods in Molecular Biology_ **In Press**.         
    ( Journal | PubMed | PDF )  

<br/>

*   [Comparative structural dynamic analysis of GTPases](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006364).  
    Li, Yao and **Grant** (2018)  
    _PLoS Comput Biol._ 14(11): e1006364.         
    ( [Journal](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006364) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30412578) | [PDF](https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.1006364&type=printable) )  

<br/>

*   [A posttranslational modification of the mitotic kinesin Eg5 that enhances its mechanochemical coupling and alters its mitotic function](https://doi.org/10.1073/pnas.1718290115).  
    Muretta, Reddy, **Scarabelli**, Thompson, **Jariwala**, Major, Venere, Rich,Willard, Thomas, Stumpff, **Grant**, Gross and Rosenfeld (2018)  
    _PNAS._ 115(8): E1779-E1788.    
    ( [Journal](https://doi.org/10.1073/pnas.1718290115) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29432173) | [PDF](http://www.pnas.org/content/early/2018/01/31/1718290115.full.pdf) )  

<br/>

*   [Altered chemomechanical coupling causes impaired motility of the kinesin-4 motors KIF27 and KIF7](http://jcb.rupress.org/content/217/4/1319.long).  
    Yue, Blasius, Zhang, **Jariwala**, Clarence , Walker, **Grant**, Cochran and Verhey (2018)  
    _J Cell Bio._ 217(4):1319-1334.      
    ( [Journal](http://jcb.rupress.org/content/217/4/1319.long) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29351996) | [PDF](http://jcb.rupress.org/content/jcb/217/4/1319.full.pdf) )  

<br/>

*   [Navigating the conformational landscape of G protein-coupled receptor kinases during allosteric activation](http://www.jbc.org/content/early/2017/08/14/jbc.M117.807461.abstract).  
    **Yao**, Cato, Labudde, Beyett, Tesmer and **Grant**. (2017)  
    _J Biol Chem._ 292(39), 16032-16043.  
    ( [Journal](http://www.jbc.org/content/early/2017/08/14/jbc.M117.807461.abstract) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28808053) | [PDF](http://www.jbc.org/content/early/2017/08/14/jbc.M117.807461.abstract) )  

<br/>

*   [A structural model for microtubule minus-end recognition and protection by CAMSAP proteins](https://www.nature.com/articles/nsmb.3483).  
    Atherton, Jiang, Stangier, Luo, Hua, Houben,  van Hooff, Joseph, **Scarabelli**, **Grant**, Roberts, Topf, Steinmetz, Baldus, Moores, Akhmanova (2017)  
    _Nature Structural & Molecular Biology._ 24, 931–943.   
    ( [Journal](https://www.nature.com/articles/nsmb.3483) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28991265) | PDF )

<br/>

*   [Structural and molecular mechanisms of cytokine-mediated endocrine resistance in human breast cancer cells](http://www.sciencedirect.com/science/article/pii/S1097276517301223).  
    Stender, Nwachukwu, Kastrata, Kim, Strid, Yakir, Srinivasan, Nowak, Izard, Erumbi, Carlson, Katzenellenbogen, **Yao**,  **Grant**, Leong, Lin, Frasor, Nettles, Glass. (2017)  
    _Molecular Cell_. 65(6), 1122-1135.  
    ( [Journal](http://www.sciencedirect.com/science/article/pii/S1097276517301223) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28306507) | [PDF](http://ac.els-cdn.com/S1097276517301223/1-s2.0-S1097276517301223-main.pdf) )  

<br/>

*   [Investigating protein sequence-structure-dynamics relationships with Bio3D-web](https://www.jove.com/video/55640/).  
    **Jariwala**, Skjærven, **Yao**,  **Grant**. (2017)  
    _Journal of Visualized Experiments (JoVE)_. Jul 16;(125). doi: 10.3791/55640.    
    ( [Journal](https://www.jove.com/video/55640/) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28745621) | [Video Article](https://www.jove.com/video/55640/investigating-protein-sequence-structure-dynamics-relationships-with) | [PDF](https://www.jove.com/pdf/55640/jove-protocol-55640-investigating-protein-sequence-structure-dynamics-relationships-with) )   


<br/>

*   [Computational and biochemical characterization of two partially overlapping interfaces and multiple weak-affinity K-Ras dimers](http://www.nature.com/articles/srep40109).  
    Prakash, Sayyed-Ahmad, Cho, Dolino, Chen, **Li**, **Grant**, Hancock, Gorfe. (2017)  
    _Scientific Reports_. 7, 40109 doi:10.1038/srep40109.  
    ( [Journal](http://www.nature.com/articles/srep40109) | [PubMed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5220301/) | [PDF](http://www.nature.com/articles/srep40109.pdf) )   

<br/>

*   [Systems structural biology analysis of ligand effects on ERα predicts cellular response to environmental estrogens and anti-hormone therapies](https://dx.doi.org/10.1016/j.chembiol.2016.11.014).  
    Nwachukwu, Srinivasan, Bruno, Nowak, Wright, Minutolo, Erumbi, Izard, **Yao**,  **Grant**, Kojetin, Elemento, Katzenellenbogen, Nettles. (2017)  
    _Cell Chemical Biology_. 24(1), 35-45.   
    ( [Journal](https://dx.doi.org/10.1016/j.chembiol.2016.11.014) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28042045) | [PDF](https://dx.doi.org/10.1016/j.chembiol.2016.11.014) )   

<br/>



Access our <a href="/publications/">complete publication list &rarr;</a>

