---
layout: page
title: Research
---

Research in the Grant lab focuses on the development and application of state-of-the-art computational and theoretical techniques to investigate the structure, dynamics and interactions of biomedically important proteins.

At the interface of biochemistry, biophysics and bioinformatics the unique insight gained from our research is tightly coupled to a wide range of biochemical and biophysical experiments. 

<img src="{{ "/img/Fig2_venn_whiteBCK.png" | prepend: site.baseurl }}" alt="Research Intersection" style="float:left;margin-right: 15px;margin-top: 10px;heigth:245px;width:245px" >

Using this approach we have recently developed a new class of allosteric Ras & Rho molecular switch inhibitors, reported the first rationally designed faster velocity kinesin motors, demonstrated for the first time that kinesin processivity can be modulated by rational mutation of the motor domain, discovered key atomistic detriments of allosteric activation in G proteins, and developed the Bio3D software package used by thousands of researchers around the world.  

Our research goals are to advance these efforts to further facilitate the fields of protein design and drug discovery.

<br/>
<a name="motors"></a>

### [Molecular Motors & Molecular Switches](#motors)

<h5>Deciphering the physical mechanisms underlying molecular motor and switch function and how their dysfunction is related to disease.</h5>

A major portion of our work asks about the common and divergent molecular mechanisms of kinesin motors and homologous G proteins switches.  These mechanisms drive the internal coordination, communication and self-organization of eukaryotic cells; and their aberrant activities underlie a wide range of human cancers and neurodegenerative disorders.  


<img src="{{ "/img/cell.jpg" | prepend: site.baseurl }}" alt="Example switches and motors" style="float:left;margin-right:20px;margin-top:15px;height:375px" >

The molecular motors and switches that we focus on comprise a set of related nucleotide binding proteins that coordinate important cell processes including signaling, trafficking, transport, and protein synthesis. Our research aims to decipher the [structural dynamic mechanisms]() by which these fascinating natural nanomachines function and how their dysfunction is related to disease.  

**Our recent findings on these systems include how to rationally engineer faster velocity longer running molecular motors and how to both allosterically activate and inhibit molecular switches that are malfunctioning in more than 20% of all human cancers.**


<a href="/docs/usage/">Read more &rarr;</a>

<br/>
<a name="bioinf"></a>

### [Methods for Structural Bioinformatics & Computer Aided Drug Design](#bioinf)

<h5>Integrating protein structural dynamics, evolutionary analysis & structure based drug discovery.</h5>

Proteins are structurally flexible molecules that often depend on their coordinated dynamic fluctuations to function properly. However, most current drug development strategies take one or a few rigid protein structures and design a single molecule to block its activity (*i.e.*, catalytic pocket). We are developing a more sophisticated methodology that exploits flexible target structures, allosteric pocket identification and evolutionary analysis. 

<iframe src="https://player.vimeo.com/video/27214521?autoplay=1&loop=1&title=0&byline=0&portrait=0" width="800" height="235" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>

To this aim, our lab introduced [Bio3D](http://thegrantlab.org/bio3d/index.php), a widely used open source R package for the analysis of protein sequence-structure-dynamics relationships. In particular, Bio3D is widely used by researchers around the world for combining information from all available experimental structures and sequences with that predicted by theory and computations.  

<img src="http://thegrantlab.org/research/files/still.jpg" alt="Prediction of drugable sites on Ras" style="float:right;margin-left: 10px;margin-top: 15px" >
Motivated by the wide utility and use of Bio3D, together with our own research questions, we are currently working to extend functionality and interoperability with leading cheminformatics resources. This will enable the inclusion of protein structural flexibility and allosteric pocket detection into structure based drug design protocols. 

<h5>Merging these new methods in a single integrated environment represents an important advance that will greatly facilitate both  bioinformatics research and the design of selective inhibitors.</h5>

<a href="http://thegrantlab.org/bio3d/index.php">Read more &rarr;</a>


<br/>
<a name="pubs"></a>

### [Recent Publications](#pubs)

*   [Rapid Characterization of Allosteric Networks with Ensemble Normal Mode Analysis](https://www.ncbi.nlm.nih.gov/pubmed/27056373).  
    Yao, Skjærven, **Grant**. (2016)  
    _J Phys Chem B._ 2016 Apr 20. [Epub ahead of print]  
    ( [Journal](http://pubs.acs.org/doi/abs/10.1021/acs.jpcb.6b01991) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27056373) | [PDF](http://pubs.acs.org/doi/pdf/10.1021/acs.jpcb.6b01991) )   

<br/>

*   [Dynamic coupling and allosteric networks in the alpha subunit of heterotrimeric G proteins](http://www.ncbi.nlm.nih.gov/pubmed/26703464).  
    Yao, Malik, Griggs, Skjaerven, Traynor, Sivaramakrishnan, **Grant**. (2016)  
    _J Biol Chem._ 291(9), 4742-4753.  
    ( [Journal](http://www.jbc.org/content/291/9/4742) | [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/26703464?dopt=Abstract) | [PDF](http://www.jbc.org/content/291/9/4742.full.pdf) )   

<br/>

*   [Mapping the processivity determinants of the kinesin-3 motor domain](https://www.ncbi.nlm.nih.gov/pubmed/26488644).  
    Scarabelli, Soppina, Yao, Atherton, Moores, Verhey, **Grant**. (2015)  
    *Biophys J.* 109, 1537–1540.  
    ( [Journal](http://www.sciencedirect.com/science/article/pii/S0006349515008620) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/26488644) | [PDF](http://www.sciencedirect.com/science/article/pii/S0006349515008620/pdfft?md5=ef9b825e43ee7400e983e52047c2c3cb&pid=1-s2.0-S0006349515008620-main.pdf) )  

<br/>

*   [Enrichment of druggable conformations from apo protein structures using cosolvent-accelerated molecular dynamics](https://www.ncbi.nlm.nih.gov/pubmed/25906084).  
    Kalenkiewicz, **Grant**, Yang. (2015)  
    *Biology.* 4, 344-366  
    ( [Journal](http://www.mdpi.com/2079-7737/4/2/344) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25906084) | [PDF](http://www.mdpi.com/2079-7737/4/2/344/pdf) )  

<br/>

*   [Integrating protein structural dynamics and evolutionary analysis with Bio3D](https://www.ncbi.nlm.nih.gov/pubmed/25491031).  
    Skjærven, Yao, Scarabelli, **Grant**. (2014)  
    *BMC Bioinformatics.* 15, 399  
    ( [Journal](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-014-0399-6) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25491031) | [PDF](http://download.springer.com/static/pdf/978/art%253A10.1186%252Fs12859-014-0399-6.pdf?originUrl=http%3A%2F%2Fbmcbioinformatics.biomedcentral.com%2Farticle%2F10.1186%2Fs12859-014-0399-6&token2=exp=1462210115~acl=%2Fstatic%2Fpdf%2F978%2Fart%25253A10.1186%25252Fs12859-014-0399-6.pdf*~hmac=fd043fd487d83e435068b0386adb21c143fc83bfee4a16f83303547a3bd16253) )  

<br/>

*   [Kinesin-5 allosteric inhibitors uncouple the dynamics of nucleotide,  microtubule and neck-linker binding sites](https://www.ncbi.nlm.nih.gov/pubmed/25418105).  
    Scarabelli, **Grant**. (2014)  
    *Biophys J.* 107, 2204–2213  
    ( [Journal](http://www.sciencedirect.com/science/article/pii/S0006349514009588) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25418105) | [PDF](http://www.sciencedirect.com/science/article/pii/S0006349514009588/pdfft?md5=8a404879535cfbf464ce2e8610472011&pid=1-s2.0-S0006349514009588-main.pdf) )  

<br/>

*   [Mapping the structural and dynamical features of kinesin motor domains](https://www.ncbi.nlm.nih.gov/pubmed/24244137).  
    Scarabelli, **Grant**. (2013)  
    *PLoS Comput Biol.* 9, e1003329  
    ( [Journal](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003329) | [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/24244137) | [PDF](http://journals.plos.org/ploscompbiol/article/asset?id=10.1371%2Fjournal.pcbi.1003329.PDF) )   

<br/>

*   [Domain opening and dynamic coupling in the alpha subunit of heterotrimeric G proteins](http://www.ncbi.nlm.nih.gov/pubmed/23870276).  
    Yao, **Grant**. (2013)  
    *Biophys J.* 105, L08–L10  
    ( [Journal](http://www.sciencedirect.com/science/article/pii/S0006349513006802) | [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/23870276) | [PDF](http://www.sciencedirect.com/science/article/pii/S0006349513006802/pdfft?md5=a387ba75bf94b47d96e3dcce464c2740&pid=1-s2.0-S0006349513006802-main.pdf) )  

<br/>

Access our <a href="/publications/">complete publication list &rarr;</a>

