---
layout: page
title: Research Overview
---

**Research in the [Grant Lab](http://thegrantlab.org) applies bioinformatics to important questions in cell biology, molecular biology, biophysics and biochemistry.** This includes investigating molecular mechanism of kinesin motors and the microtubule cytoskeleton; signal transduction through G proteins, membrane receptors and channels; the evolution of functional specialization in large protein superfamilies; and characterizing the atomistic basis of protein allosteric mechanisms. 

<iframe src="https://player.vimeo.com/video/10426396?loop=1&color=ffffff&byline=0&portrait=0&autoplay=1&title=1" width="320" height="240" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
<iframe src="https://player.vimeo.com/video/24146793?loop=1&color=ffffff&byline=0&portrait=0" width="470" height="240" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>

<br/>

### [GeneralApproach]()

Our general approach is to develop and apply state-of-the-art computational techniques at multiple levels that can be coupled to a wide range of biochemical and biophysical experiments. This includes:

- **Bioinformatic analysis** to probe sequence-structure-function relationships; 
- **Molecular dynamics** to investigate essential conformational changes; 
- **Brownian dynamics** for diffusional protein-protein and protein-ligand encounters; 
- **Computer-aided drug design** for discovering novel therapeutics; and 
- **Genome informatics** for mapping SNPs and disease associated variants to structural dynamic consequences and potential functional effects.  

<br/>

### [Major Contributions]()

In this way we have gained a number of unique insights and made important contributions. This includes the development of a new class of allosteric Ras & Rho molecular switch inhibitors [(1, 2)](#pubs), reporting the first rationally designed faster velocity kinesin motors [(3)](#pubs) , demonstrating for the first time that kinesin processivity can be modulated by rational mutation of the motor domain [(4)](#pubs), discovery of key atomistic determinants of allosteric activation in G proteins [(5)](#pubs), successfully predicting the mechanism of a major immunologically mediated adverse drug reaction [(6)](#pubs), discovery of disease associated mutations and post-translational modifications that can tune kinesin protein function [(7)](#pubs), and development of the Bio3D software package used by thousands of researchers and students around the world [(8-10)](#pubs).   

Collectively, this broad body of work demonstrates the power of applying a bioinformatics informed approach to biology. **Our research goals are to advance these efforts to further facilitate the fields of protein design and drug discovery.** 

<br/>

<a name="pubs"></a>

### [Selected References]()

1.  Grant, B. J., S. Lukman, H. J. Hocker, J. Sayyah, J. H. Brown, J. A. McCammon, and A. A. Gorfe. 2011. Novel allosteric sites on Ras for lead generation. PLoS One 6:e25711.

2.  Ortiz-Sanchez, J. M., S. E. Nichols, J. Sayyah, J. H. Brown, J. A. McCammon, and B. J. Grant. 2012. Identification of potential small molecule binding pockets on Rho family GTPases. PLoS One 7:e40809.

3.  Grant, B. J., D. M. Gheorghe, W. Zheng, M. Alonso, G. Huber, M. Dlugosz, J. A. McCammon, and R. A. Cross. 2011. Electrostatically biased binding of kinesin to microtubules. PLoS Biol 9:e1001207.

4.  Scarabelli, G., V. Soppina, X. Q. Yao, J. Atherton, C. A. Moores, K. J. Verhey, and B. J. Grant. 2015. Mapping the Processivity Determinants of the Kinesin-3 Motor Domain. Biophys J 109:1537-1540.

5.  Yao, X. Q., R. U. Malik, N. W. Griggs, L. Skjaerven, J. R. Traynor, S. Sivaramakrishnan, and B. J. Grant. 2016. Dynamic Coupling and Allosteric Networks in the alpha Subunit of Heterotrimeric G Proteins. J Biol Chem 291:4742-4753.

6.  Ostrov, D. A., B. J. Grant, Y. A. Pompeu, J. Sidney, M. Harndahl, S. Southwood, C. Oseroff, S. Lu, J. Jakoncic, C. A. de Oliveira, L. Yang, H. Mei, L. Shi, J. Shabanowitz, A. M. English, A. Wriston, A. Lucas, E. Phillips, S. Mallal, H. M. Grey, A. Sette, D. F. Hunt, S. Buus, and B. Peters. 2012. Drug hypersensitivity caused by alteration of the MHC-presented self-peptide repertoire. Proc Natl Acad Sci U S A 109:9959-9964.

7.  Muretta, J., B. J. Reddy, G. Scarabelli, B. J. Grant, S. P. Gross, and S. S. Rosenfeld. 2016. Allosterically tuning the mechanochemical properties of kinesin-5 via mutations and post translational modifications. In preparation.

8.  Grant, B. J., A. P. Rodrigues, K. M. ElSawy, J. A. McCammon, and L. S. Caves. 2006. Bio3D: an R package for the comparative analysis of protein structures. Bioinformatics 22:2695-2696.

9.  Skjaerven, L., X. Q. Yao, G. Scarabelli, and B. J. Grant. 2014. Integrating protein structural dynamics and evolutionary analysis with Bio3D. BMC Bioinformatics 15:399.

10. Skjaerven, L., S. Jariwala, X. Q. Yao, and B. J. Grant. 2016. Online interactive analysis of protein structure ensembles with Bio3D-web. Bioinformatics 2016;32(22):3510-3512.


[Additional Grant Lab Publications &rarr;]({{ "/publications/" | prepend: site.baseurl }})



